tests/Timer/TimersTest.php

```diff
@@ -24,4 +24,17 @@ public function testBlockedTimer()
 
         $this->assertTrue(true);
     }
+
+    public function testContains()
+    {
+        $timers = new Timers();
+
+        $timer1 = new Timer(0.1, function () {});
+        $timer2 = new Timer(0.1, function () {});
```

Comment:
LGTM as-is, but for the reference and because I'm *sure* you're going to appreciate this: Legacy PHPUnit does not support `$this->createMock('React\EventLoop\TimerInterface')`, but this can *easily* be worked around with `$this->createMock('React\EventLoop\TimerInterface')->getMock()`. :wink:

Reply:
Yeah, I avoided the mock builder and went with the real instance. Would've used an anonymous class if this was 7.0+